code/game/gamemodes/miniantags/hades/hades.dm

```diff
@@ -0,0 +1,247 @@
+							for(var/mob/living/carbon/human/H in player_list) // name lottery
+								if(H == sinPerson)
+									continue
+								if(prob(25))
+									spawn(10)
+										sinPerson.name = H.name
+										sinPerson.real_name = H.real_name
+										var/datum/dna/lottery = H.dna
+										lottery.transfer_identity(sinPerson, transfer_SE=1)
+										sinPerson.updateappearance(mutcolor_update=1)
+										sinPerson.domutcheck()
+						else
+							src.say("Your sin will be punished, [sinPerson]!")
+							var/sinPersonspecies = pick(species_list)
+							var/newtype = species_list[sinPersonspecies]
+							var/datum/species/old_species = sinPerson.dna.species
+							sinPerson.set_species(newtype)
+							sinPerson.dna.species.admin_set_species(sinPerson,old_species)
+					if("Sloth")
+						src.say("Your sin, [sinPerson], is Sloth.")
+						if(prob(50))
+							src.say("I will indulge your sin, [sinPerson].")
+							sinPerson.drowsyness += 50
+						else
+							src.say("Your sin will be punished, [sinPerson]!")
+							sinPerson.reagents.add_reagent("frostoil", 50)
+					if("Wrath")
+						src.say("Your sinPerson, [sinPerson], is Wrath.")
+						if(prob(50))
+							src.say("I will indulge your sin, [sinPerson].")
+							sinPerson.reagents.add_reagent("bath_salts",100)
+						else
+							src.say("Your sin will be punished, [sinPerson]!")
+							sinPerson.reagents.add_reagent("lexorin", 100)
+							sinPerson.reagents.add_reagent("mindbreaker", 100)
+
+
+///Sin related things
+/obj/item/weapon/twohanded/sin_pride
+	icon_state = "mjollnir0"
+	name = "Pride-struck Hammer"
+	desc = "It resonates an aura of Pride."
+	force = 5
+	throwforce = 15
+	w_class = 4
+	slot_flags = SLOT_BACK
+	force_unwielded = 8
+	force_wielded = 18
+	attack_verb = list("attacked", "smashed", "crushed", "splattered", "cracked")
+	hitsound = 'sound/weapons/blade1.ogg'
+	var/pride_direction = 0 // 0 owner > target; 1 target > owner
+
+/obj/item/weapon/twohanded/sin_pride/update_icon()  //Currently only here to fuck with the on-mob icons.
+	icon_state = "mjollnir[wielded]"
+	return
+
+/obj/item/weapon/twohanded/sin_pride/afterattack(atom/A as mob|obj|turf|area, mob/user, proximity)
+	if(!proximity) return
+	if(wielded)
+		if(istype(A,/mob/living/carbon/human))
+			var/mob/living/carbon/human/H = A
+			if(H)
+				if(pride_direction == 0)
+					user.reagents.trans_to(H, user.reagents.total_volume, 1, 1, 0)
+					user << "Your pride reflects on [H]."
+					H << "You feel insecure, taking on [user]'s burden."
+				else if(pride_direction == 1)
+					H.reagents.trans_to(user, H.reagents.total_volume, 1, 1, 0)
+					H << "Your pride reflects on [user]."
+					user << "You feel insecure, taking on [H]'s burden."
+
+/obj/item/lovestone
+	name = "Stone of Lust"
+	desc = "It lays within your hand, radiating pulses of uncomfortable warmth."
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "lovestone"
+	item_state = "lovestone"
+	w_class = 1
+	var/lust_direction = 0

```

Comment:
This var name is balls. Also, it uses 0 and 1 to determine its behaviour despite being undescriptive as fuck.